pacemaker.spec.in

```diff
@@ -630,6 +653,7 @@ exit 0
 
 %config(noreplace) %{_sysconfdir}/sysconfig/pacemaker
 %if %{defined _unitdir}
+%ghost %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
%ghost will only remove a directory if it's empty, so maybe %ghost the file, too (though I guess that would only matter in the unlikely case the package is removed after an interrupted upgrade)